Turtle parser fails when a store without formula support is used

The Turtle parser fails with the following error when a `Store` without formula support is used (i.e. with `formula_aware = False`).

Errror:
```
    g.parse("test.ttl", format="ttl")
venv/lib/python3.8/site-packages/rdflib/graph.py:1549: in parse
    context.parse(source, publicID=publicID, format=format, **args)
venv/lib/python3.8/site-packages/rdflib/graph.py:1078: in parse
    parser.parse(source, self, **args)
venv/lib/python3.8/site-packages/rdflib/plugins/parsers/notation3.py:1884: in parse
    p = SinkParser(sink, baseURI=baseURI, turtle=turtle)
venv/lib/python3.8/site-packages/rdflib/plugins/parsers/notation3.py:417: in __init__
    self._formula = store.newFormula()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <rdflib.plugins.parsers.notation3.RDFSink object at 0x7fb6fe2f1a60>

    def newFormula(self):
>       assert self.graph.store.formula_aware
E       AssertionError

venv/lib/python3.8/site-packages/rdflib/plugins/parsers/notation3.py:1712: AssertionError
```

Code:
```python
g = Graph(store="OxMemory")
g.parse("test.ttl", format="ttl")
```

The store is provided by [oxrdflib](https://github.com/oxigraph/oxrdflib).

--- a/rdflib/plugins/parsers/notation3.py
+++ b/rdflib/plugins/parsers/notation3.py
@@ -1726,6 +1726,7 @@ r_hibyte = re.compile(r"([\x80-\xff])")
 class RDFSink(object):
     def __init__(self, graph):
         self.rootFormula = None
+        self.uuid = uuid4().hex
         self.counter = 0
         self.graph = graph
 
@@ -1745,7 +1746,7 @@ class RDFSink(object):
             return arg.newBlankNode(uri)
         elif isinstance(arg, Graph) or arg is None:
             self.counter += 1
-            bn = BNode("n" + str(self.counter))
+            bn = BNode("n%sb%s" % (self.uuid, self.counter))
         else:
             bn = BNode(str(arg[0]).split("#").pop().replace("_", "b"))
         return bn

--- a/rdflib/plugins/parsers/notation3.py
+++ b/rdflib/plugins/parsers/notation3.py
@@ -404,7 +404,7 @@ class SinkParser:
             else:
                 self._genPrefix = uniqueURI()
 
-        if openFormula is None:
+        if openFormula is None and not turtle:
             if self._thisDoc:
                 self._formula = store.newFormula(thisDoc + "#_formula")
             else:

--- a/test/test_issue1141.py
+++ b/test/test_issue1141.py
@@ -1,0 +1,27 @@
+import unittest
+from io import BytesIO
+
+from rdflib import Graph
+from rdflib.plugins.memory import IOMemory
+from rdflib.plugins.stores.auditable import AuditableStore
+
+
+class TestIssue1141(unittest.TestCase):
+    """
+    Tests is Turtle and TriG parsing works with a store with or without formula support
+    """
+    def test_issue_1141(self):
+        file = b"@prefix : <http://example.com/> . :s :p :o ."
+
+        for format in ("turtle", "trig"):
+            # with formula
+            graph = Graph()
+            self.assertTrue(graph.store.formula_aware)
+            graph.load(BytesIO(file), format=format)
+            self.assertEqual(len(graph), 1)
+
+            # without
+            graph = Graph(store=AuditableStore(IOMemory()))
+            self.assertFalse(graph.store.formula_aware)
+            graph.load(BytesIO(file), format=format)
+            self.assertEqual(len(graph), 1)

